src/classes/view/video.vala

```diff
@@ -27,28 +27,30 @@ namespace pdfpc {
      * video.
      */
     public class View.Video: Gtk.Fixed {
-        private Gtk.Widget video = null;
+        private List<Gtk.Widget> videos = null;
+
+        public Video() {
+            this.videos = new List[Gtk.Widget]();
```

Comment:
`new List<Gtk.Widget>();`

?

Reply:
Ah sorry, I am not very familiar with Vala and missed it. I am surprised that the code compiled in the first place.